README.md

```diff
@@ -70,6 +70,10 @@ Visit [RBAC permissions for running the tasks](docs/tasks-rbac-permissions.md) i
 
 - [wait-for-vmi-status](tasks/wait-for-vmi-status)
 
+#### Modify Windows iso
+- [modify-windows-iso-file](tasks/modify-windows-iso-file) - modifies windows iso (replaces prompt bootloader with no-promt 
```

Comment:
Can you explain why one should use the noprompt bootloader?

Reply:
That is explained in readme of the task https://github.com/kubevirt/kubevirt-tekton-tasks/pull/219/files#diff-8cb4772fdd977a741135b0b25c759fd92903ecf1121237e85f4a5e345ea95922